app/javascript/components/widget-bar.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { colors } from '../lib/theme';
+
+export const CurrentWidgetBar = ({ time }) => (
+  <svg width="568" height="4">
+    <rect width={568 * (time / 20000)} height="4" fill={colors.white} />
+    <rect width="568" height="1" fill={colors.white} />
+  </svg>
+);
+
+export const OtherWidgetBar = () => (
```

Comment:
Is there a more specific word we can use than `Other`? Or a pair of adjectives we can use instead (Active/Inactive)?

Reply:
I like Active and Inactive